src/core/bond_exist.cpp

```diff
@@ -0,0 +1,22 @@
+#include"bond_exist.hpp"
+#include"interaction_data.hpp"
+#include"bond.hpp"
+bool make_bond_exist_bond_class(int bond_type){
```

Comment:
The move semantics is needed because unique_ptr can not be copied. (it would not be very unique otherwise...)